cmd/ui/main.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"os/signal"
+	"syscall"
+
+	"github.com/abcxyz/jvs/pkg/ui"
+	"github.com/abcxyz/pkg/logging"
+)
+
+func main() {
```

Comment:
Please use the `realMain` pattern so that defers and cleanups run.

Reply:
ack, i referenced the main.go in `jvsctl` but I see the other pattern.